fontforgeexe/startui.c

```diff
@@ -1248,12 +1250,15 @@ int fontforge_main( int argc, char **argv ) {
     /*  the window around, which I can determine if I have a positioned */
     /*  decorated window created at the begining */
     /* Actually I don't care any more */
-    wattrs.mask = wam_events|wam_cursor|wam_bordwidth|wam_backcol|wam_positioned|wam_utf8_wtitle|wam_isdlg;
+    /* happyrobotstudio Jul_8_2019: updated to re-include wam_nodecor and wattrs.nodecoration = 1 */
+
+    wattrs.mask = wam_events|wam_cursor|wam_bordwidth|wam_backcol|wam_positioned|wam_nodecor; //|wam_utf8_wtitle|wam_isdlg;
```

Comment:
These comments aren't really needed&hellip`git blame` is good enough. Also, please refrain from making a new PR when changes are requested, please push your changes to the initial PR.

Reply:
Hiya,

Updated this item, 

Initially i was working on happyrobotstudio:fontforge:master

However thought to create a Branch in the usual fashion .. will update same branch in future

Cheers!